style.css

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.cdnfonts.com/css/Roboto');
+@import url('https://fonts.cdnfonts.com/css/poppins');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+.top_header {
+  background: #3c3a39;
+  width: 375px;
+}
+
+.top_nav {
+  color: #fff;
+  display: flex;
```

Comment:
* [ ]  It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should slightly_smiling_face. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
 
 ```
 .flex-row {
   display: flex;
 }
 
 .flex-column {
   display: flex;
   flex-direction: column;
 }
 ```
 
 * And kindly attend to any repetitions in your code :slightly_smiling_face:

Reply:
@Efiamotu-1 ,  
Thanks, I have never thought of it this way,  grateful for the insight.
this was resolved for the flex_row/column, I could not find any other directly repeated items that can be split without making the CSS extra long and complex. for instance, I have repeated font-family: toppings, but would it make sense to have a class for font-family?.